src/clr-angular/accordion/accordion-panel.html

```diff
@@ -25,8 +24,10 @@
           <clr-icon shape="check-circle" class="clr-accordion-complete-icon"></clr-icon>
         </span>
         <ng-content select="clr-accordion-title, clr-step-title"></ng-content>
+        <div #description>
```

Comment:
Block-level elements are not allowed inside a button element. But if resort to `ContentChild` on `ClrAccordionTitle`/`ClrAccordionDescription` as I mentioned above, we could avoid this extra wrapper element altogether.